src/operation/problem_results.jl

```diff
@@ -35,7 +45,7 @@ function ProblemResults(model::DecisionModel)
     duals = read_duals(container)
     parameters = read_parameters(container)
     timestamps = get_timestamps(model)
-    optimizer_stats = OptimizerStats(model)
+    optimizer_stats = to_dataframe(OptimizerStats(model))
```

Comment:
In order to have type consistency this struct built from a DecisionModel will have a DataFrame with one row. If it's built from an EmulationModel then it will have one row per execution. We might need to add something else to the struct to indicate how it was built.

Reply:
We could add something to problem results to indicate that it was a DecisionModel or en EmulationModel.

Reply:
Record the type as a string in the struct.